pipeline/update_db.py

```diff
@@ -23,25 +35,63 @@ def get_db_connection():
     return eng
 
 
-def update_table(data, table, eng):
+def update_games(data, eng):
+    with eng.begin() as conn:
+        for _, row in data.iterrows():
+            stmt = f"SELECT * FROM {GAMES_TABLE} WHERE id = {row['id']};"
+            game = conn.execute(stmt).first()
+            # if game not in DB, insert all values and move on
+            if not game:
+                values = tuple(row[GAME_COLS])
+                stmt = f"INSERT INTO {GAMES_TABLE} VALUES {values};"
+                conn.execute(stmt)
+                continue
+            # if postponed/suspended, flag as date changed and nullify date
+            if row["date_changed"]:
+                stmt = (
+                    f"UPDATE {GAMES_TABLE} "
+                    f"SET date_changed = True, date = null "
+                    f"WHERE id = {row['id']};"
+                )
+                conn.execute(stmt)
+            # for games already in DB, update date/week/doubleheader
+            dt, wk, dh = row["date"], row["week"], row["dh"]
```

Comment:
If not, then we should compare `row["date"]` to `game.date` and only perform the update if they don't match (adding `date_changed = True` to the update as well)

Reply:
I'm not sure that can happen --- if a game has been wagered it must have already been returned from the API. Then if it gets rescheduled, it will have to appear again with a status of postponed, right?

Either way your suggestion is more robust, gonna go with that!

Reply:
Yeah I'm just thinking about weird edge cases.

I have a vague recollection of the schedule API documentation having several different variations on game statuses that might change unexpectedly